azure/config/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM mcr.microsoft.com/azureml/base-gpu:openmpi3.1.2-cuda10.1-cudnn7-ubuntu18.04
+ 
+RUN apt-get update -y && \
+    apt-get install -y curl
+ 
+RUN ldconfig /usr/local/cuda/lib64/stubs && \
+    conda install -y conda=4.6.14 python=3.6.2 && conda clean -ay && \
+    # Install AzureML SDK
+    pip install --no-cache-dir azureml-defaults && \
```

Comment:
This is probably not needed for `torchelastic`?

Reply:
This installs the AzureML SDK on the cluster.